tests/openwisp2/celery.py

```diff
@@ -1,11 +1,11 @@
-from __future__ import absolute_import, unicode_literals
-
 import os
 
 from celery import Celery
 
+from django.conf import settings
+
 os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'openwisp2.settings')
 
 app = Celery('openwisp2')
-app.config_from_object('django.conf:settings', namespace='CELERY')
-app.autodiscover_tasks()
+app.config_from_object('django.conf:settings', namespace = 'CELERY')
+app.autodiscover_tasks(settings.INSTALLED_APPS)
```

Comment:
revert the changes on these two lines please, see also:
https://github.com/celery/celery/issues/3341#issuecomment-235408048